Formula/votca.rb

```diff
@@ -0,0 +1,40 @@
+class Votca < Formula
+  desc "Versatile Object-oriented Toolkit for Coarse-graining Applications"
+  homepage "https://www.votca.org/"
+  url "https://github.com/votca/votca/archive/refs/tags/v2023-rc.1.tar.gz"
```

Comment:
We do not accept pre-release versions.

Reply:
Ok, I will push out the real release next week.